File: Questions/sieve.js
let list = new Array(101)
for(let i=0; i<list.length; i++) list[i] = true;
list[0] = list[1] = false
for(let i=2; i<list.length; i++){
    if(list[i]){
        for(let j=2*i; j<list.length; j+=i)
        list[j] = false
    }
}
for(let i=2; i<list.length; i++){
    if(list[i]){
        console.log(`${i} is a prime number.`)
    }
}

File: Questions/Sieve.py
list = []
for i in range(101):
    list.append(True)
list[0] = list[1] = False


for i in range(2,len(list)):
    if list[i]==True:
        for j in range(2*i, len(list),i):
            list[j]=False

for idx, e in  enumerate(list):
    if e:
        print(idx, " is a prime number.")